graylog2-web-interface/src/views/components/NewWidgetPlaceholder.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useCallback, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import type WidgetPosition from 'views/logic/widgets/WidgetPosition';
+import { Icon } from 'components/common';
+
+const PlaceholderBox = styled.div(({ theme }) => css`
+  opacity: 0;
+  transition: visibility 0s, opacity 0.2s linear;
+  
```

Comment:
Let's add `cursor: pointer` to this element, just to make it even more clear that you can click everywhere and not only on the icon.